ccd-definition/CaseEvent/User/UserEventsCaseProg-nonprod.json

```diff
@@ -67,6 +67,22 @@
     "ShowEventNotes": "N",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "TRIAL_READINESS",
+    "Name": "Trial Readiness",
+    "Description": "Trial Readiness",
```

Comment:
Is the post & pre state supposed to be the same ?

Reply:
[CIV-6051](https://tools.hmcts.net/jira/browse/CIV-6051) says:
Event should be available within the following states in [Case Progression - MVP Requirements - State and Event Model](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1593580315#CaseProgressionMVPRequirements-StateandEventModel):
PREPARE_FOR_HEARING_CONDUCT_HEARING

I'm assuming that within means that pre and post conditions are the same.

Reply:
No, this is the pre state. Speak to Doug about post state requirement

Reply:
I talked to Doug, and he said that this is correct.